yarn-project/noir-contracts/src/contracts/docs_example_contract/src/actions.nr

```diff
@@ -1,22 +1,13 @@
-use dep::std::option::Option;
-use dep::aztec::constants_gen::{MAX_READ_REQUESTS_PER_CALL, MAX_NOTES_PER_PAGE};
-use dep::aztec::context::{
-    PrivateContext,
-};
+use dep::aztec::constants_gen::{MAX_NOTES_PER_PAGE, MAX_READ_REQUESTS_PER_CALL};
```

Comment:
Did you run a code formatter or something? If so, how? We should get it into the CI so we ensure all our code gets formatted.

Reply:
FWIW I found a tracking issue in Noir, which doesn't seem prioritised atm: https://github.com/noir-lang/noir/issues/1580

As for using rustfmt, it's an interesting idea, but I'm worried that it'll keep diverging as we add more non-rust features to Noir. Also, removing noir-specific keywords is easy, but adding them back is not, given the file changes introduced by rustfmt. I think we should probably wait.